controller/tokenization_manager.py

```diff
@@ -0,0 +1,206 @@
+from typing import Any, Dict, List, Tuple
+from controller.tokenizer import add_attribute_to_docbin, tokenize_record
+import traceback
+from controller.task_util import (
+    finalize_task,
+    set_task_to_started,
+    update_tokenization_progress,
+)
+from submodules.model.models import Attribute, RecordTokenizationTask
+from misc.notification import send_notification_created
+from submodules.model import enums
+from submodules.model.business_objects import (
+    attribute,
+    general,
+    notification,
+    project,
+    record,
+    tokenization,
+)
+from handler.tokenizer_handler import get_tokenizer_by_project
+from misc.util import send_websocket_update, upload_to_minio_after_every_10th_chunk
+
+__prioritized_records = {}
+
+
+def manage_calculated_attribute_tokenization(
+    project_id: str, user_id: str, task_id: str, initial_count: int, attribute_name: str
+) -> None:
+    try:
+        tokenization_task, tokenizer = __set_up_tokenization(
+            project_id, task_id, initial_count
+        )
+
+        attribute_item = attribute.get_by_name(project_id, attribute_name)
+        non_text_attributes = attribute.get_non_text_attributes(project_id).keys()
+        __check_attribute_is_text(attribute_item)
+
+        chunk_size, progress_per_chunk = __get_chunk_size_and_progress_per_chunk(
+            initial_count
+        )
+        record_tokenized_entries = record.get_attribute_data_with_doc_bins_of_records(
+            project_id, attribute_name
+        )
+        chunks = [
+            record_tokenized_entries[x : x + chunk_size]
+            for x in range(0, len(record_tokenized_entries), chunk_size)
+        ]
+        for idx, chunk in enumerate(chunks):
+            values = []
+            for record_tokenized_item in chunk:
+                values.append(add_attribute_to_docbin(tokenizer, record_tokenized_item))
```

Comment:
if there is a neat function we could just use list comprehension
something like:
`values = [add_attribute_to_docbin(tokenizer,item) for item in chunk]`

Reply:
Refactored to list comprehension